packages/main/src/ListItemStandard.ts

```diff
@@ -143,8 +187,39 @@ class ListItemStandard extends ListItem implements IAccessibleListItem {
 
 	onBeforeRendering() {
 		super.onBeforeRendering();
-		this.hasTitle = !!this.textContent;
+		this.hasTitle = !!(this.text || this.textContent);
 		this._hasImage = this.hasImage;
+
+		// Only load ExpandableText if "Normal" wrapping is used
+		if (this.wrappingType === "Normal") {
+			// If feature is already loaded (preloaded by the user), use the existing template
+			if (ListItemStandard.ExpandableTextTemplate) {
```

Comment:
For this to be preloaded the user will have to import it, so we must create a file ListItemStandardExpandableText.ts next to the template. For reference, check the `ColorPaletteMoreColors.ts` and all the places it is mentioned (docs, component-import.js)